pkg/mkverification-raw-efi/Dockerfile

```diff
@@ -23,7 +23,7 @@ ENV PKGS mtools dosfstools libarchive-tools sgdisk e2fsprogs util-linux \
     lm-sensors acpi iw libdrm hwinfo dhclient dmidecode smartmontools \
     pciutils libgcc pixman glib libvncserver musl-utils dhclient \
     qemu-system-x86_64 tpm2-tss-dev tpm2-tss-esys tpm2-tss-fapi tpm2-tss-rc \
-    tpm2-tss-sys tpm2-tss-tctildr tpm2-abrmd
```

Comment:
Years ago when we looked at including python (for LISP RFC) it seemed to add 60Mb to the image. What is the impact of adding this now?

Reply:
I need Python to run vmcap, so I will not include it in our installed packages. In any case, I will report the increase in space required due to Python.